src/python_testing/TC_CLCTRL_4_1.py

```diff
@@ -0,0 +1,1060 @@
+            TestStep("8c", "If the attribute is supported on the cluster, TH reads from the DUT the OverallTargetState attribute."),
+            TestStep("8d", "Wait until TH receives a subscription report with OverallCurrentState.Position = OpenedForVentilation."),
+            TestStep("8e", "Wait until TH receives a subscription report with MainState = Stopped."),
+            TestStep("9a", "If PD feature is not supported on the cluster, skip steps 9b to 9e."),
+            TestStep("9b", "TH sends command MoveTo with Position = MoveToPedestrianPosition."),
+            TestStep("9c", "If the attribute is supported on the cluster, TH reads from the DUT the OverallTargetState attribute."),
+            TestStep("9d", "Wait until TH receives a subscription report with OverallCurrentState.Position = OpenedForPedestrian."),
+            TestStep("9e", "Wait until TH receives a subscription report with MainState = Stopped."),
+            TestStep("10a", "TH sends command MoveTo with Position = MoveToFullyOpen."),
+            TestStep("10b", "If the attribute is supported on the cluster, TH reads from the DUT the OverallTargetState attribute."),
+            TestStep("10c", "Wait until TH receives a subscription report with OverallCurrentState.Position = FullyOpened."),
+            TestStep("10d", "Wait until TH receives a subscription report with MainState = Stopped."),
+            TestStep("11a", "TH sends command MoveTo with Position = MoveToFullyClosed."),
+            TestStep("11b", "If the attribute is supported on the cluster, TH reads from the DUT the OverallTargetState attribute."),
+            TestStep("11c", "Wait until TH receives a subscription report with OverallCurrentState.Position = FullyClosed."),
+            TestStep("11d", "Wait until TH receives a subscription report with MainState = Stopped."),
+            TestStep("12a", "If SP feature is not supported on the cluster, skip steps 12b to 12g."),
+            TestStep("12b", "TH sends command MoveTo with Speed = High."),
+            TestStep("12c", "If the attribute is supported on the cluster, TH reads from the DUT the OverallTargetState attribute."),
+            TestStep("12d", "Wait until TH receives a subscription report with OverallCurrentState.Speed = High."),
+            TestStep("12e", "TH sends command MoveTo with Speed = Low."),
+            TestStep("12f", "If the attribute is supported on the cluster, TH reads from the DUT the OverallTargetState attribute."),
+            TestStep("12g", "Wait until TH receives a subscription report with OverallCurrentState.Speed = Low."),
+            TestStep("13a", "If PT feature is not supported on the cluster, skip steps 13b to 13f."),
+            TestStep("13b", "TH sends TestEventTrigger command to General Diagnostic Cluster on Endpoint 0 with EnableKey field set to PIXIT.CLCTRL.TEST_EVENT_TRIGGER_KEY and EventTrigger field set to PIXIT.CLCTRL.TEST_EVENT_TRIGGER for MainState is Protected Test Event."),
```

Comment:
We should link to the correct test event trigger value i.e endpoint targeted and associated value for protected.

This comment applies everywhere the test event trigger is used.